src/studio/nuxeo-studio/nuxeo-studio-release-notes/2017-10-02-studio-70.md

```diff
@@ -0,0 +1,49 @@
+---
+title: October 2, 2017
+description: This version polished autocompletion in automation scripting, and improves flow between Modeler and Designer.
+tree_item_index: 999999995
+review:
+    comment: ''
+    date: '2017-10-02'
+    status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v70'}}
+This version improves flow between Designer and Modeler, polishes autocompletion in automation scripting and brings bug fixes.
+{{! /multiexcerpt}}
+
+## Improved Flow Between Studio Modeler and Designer
+One of the frequent feedbacks we received is that you want to be able to switch faster from Modeler to Designer (and vice versa of course).
+
+If you fall in that category, you will be glad to learn that buttons will now appear where appropriate to let you switch from one to the other, directly to the corresponding screen. When creating a document type for instance, configuring its layouts is now one click away.
+
+![]({{file name='configure-layouts.png'}} ?w=591,h=185,border=true)
+
+From Designer, a configure button is also available to take you back to your document type configuration.
+
+![]({{file name='configure-doctype.png'}} ?w=325,h=140,border=true)
+
+Similar options are provided for page providers and workflows.
+
+## Further Polished Automation Scripting Experience
+Autocompletion experience is one of the key points when using automation scripting. We felt like it still wasn't satisfying enough, so we tweaked it again with the goal to make it more stable and predictable.
```

Comment:
_Auto-completion is an important component in automation scripting. We felt as though it still wasn't up to scratch, so we tweaked it again, aiming to make it more stable and predictable._